Tests/Source/UserdataTests.cpp

```diff
@@ -0,0 +1,129 @@
+// https://github.com/vinniefalco/LuaBridge
+// Copyright 2022, Stefan Frings
+// SPDX-License-Identifier: MIT
+
+#include "TestBase.h"
+
+#include "LuaBridge/LuaBridge.h"
+
+class TestClass
+{
+public:
+    explicit inline TestClass(int i) : m_i(i) {}
+
+    inline int get() const { return m_i; }
+
+private:
+    int m_i;
+};
+
+int testFunctionObject(TestClass object)
+{
+    return object.get();
+}
+
+int testFunctionObjectConst(const TestClass object)
+{
+    return object.get();
+}
+
+int testFunctionRef(TestClass& object)
+{
+    return object.get();
+}
+
+int testFunctionRefConst(const TestClass& object)
+{
+    return object.get();
+}
+
+struct UserDataTest : TestBase
+{
+    void SetUp() override
+    {
+        TestBase::SetUp();
+
+        luabridge::getGlobalNamespace(L)
+            .beginClass<TestClass>("TestClass")
+            .addConstructor<void (*)(int)>()
+            .endClass()
+            .
```

Comment:
clang-format?

Reply:
@stefanfrings ?

Reply:
Clang-format doesn't change anything in these lines.